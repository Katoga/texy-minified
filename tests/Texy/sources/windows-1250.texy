Texy! is sexy!
**************

Texy! je pøevadìè textu do formátovaného HTML kódu. Díky nìmu mùžete 
psát strukturované dokumenty i bez znalosti nebo použití jazyka HTML. 
Dokument napíšete v pøehledné, èistì textové, formì a Texy! jej 
automaticky pøevede do validního (X)HTML kódu. 

Texy! je jedním z **nejkomplexnìjších** formátovaèù. Umí zpracovávat 
obrázky, odkazy, vnoøené seznamy, tabulky a má také plnou 
podporu CSS((Kaskádové styly)).


- rozdìlení velmi dlouhých slov podle èeských pravidel. Pøíklad: nejneobhospodaøovávatelnìjšími
- klikatelné adresy www.example.cz, dave@example.com (emaily jsou odolné vùèi spambotùm)
- "èeské" 'typografické' uvozovky
- pomlèka vs. spojovník: 10-15 vs. èesko-slovenský  
- pomlèka: jedna -- dvì
- typografický køížek u rozmìrù 10 x 20 
- nedìlitelné mezery u telefonních èísel +420 776 552 046
- vkládání nezalomitelných mezer za jednopísmenné pøedložky
- acronym NATO((North Atlantic Treaty Organisation))
- abbr "et al."((a další))
- náhrady(TM) nebo(R) za pøíslušné (C)entity
- šipky <- a -> a <->
- tøi teèky...
- zachování HTML entit &amp;
- a spousta dalšího :-)
