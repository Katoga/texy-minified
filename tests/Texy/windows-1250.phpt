<?php

/**
 * Test: Non-UTF-8 encoding.
 */

use Tester\Assert;

require __DIR__ . '/../bootstrap.php';


$texy = new Texy;
$texy->encoding = 'windows-1250';

Assert::matchFile(
	__DIR__ . '/expected/windows-1250.html',
	$texy->process(file_get_contents(__DIR__ . '/sources/windows-1250.texy'))
);
