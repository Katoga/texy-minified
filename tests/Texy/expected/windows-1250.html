<h1>Texy! is sexy!</h1>

<p>Texy! je pøevadìè textu do formátovaného HTML kódu. Díky nìmu
mùžete psát strukturované dokumenty i bez znalosti nebo použití jazyka
HTML. Dokument napíšete v pøehledné, èistì textové, formì a Texy! jej
automaticky pøevede do validního (X)HTML kódu.</p>

<p>Texy! je jedním z <strong>nejkomplexnìj­ších</strong> formátovaèù.
Umí zpracovávat obrázky, odkazy, vnoøené seznamy, tabulky a má také plnou
podporu <acronym title="Kaskádové styly">CSS</acronym>.</p>

<ul>
	<li>rozdìlení velmi dlouhých slov podle èeských pravidel. Pøíklad:
	nejneobhospoda­øovávatelnìjší­mi</li>

	<li>klikatelné adresy <a href="http://www.example.cz">www.example.cz</a>, <a
	href="mailto:dave&#64;example.com">dave&#64;<!-- -->example.com</a> (emaily jsou
	odolné vùèi spambotùm)</li>

	<li>„èeské“ ‚typografické‘ uvozovky</li>

	<li>pomlèka vs. spojovník: 10–15 vs. èesko-slovenský</li>

	<li>pomlèka: jedna – dvì</li>

	<li>typografický køížek u rozmìrù 10 × 20</li>

	<li>nedìlitelné mezery u telefonních èísel +420 776 552 046</li>

	<li>vkládání nezalomitelných mezer za jednopísmenné pøedložky</li>

	<li>acronym <acronym
	title="North Atlantic Treaty Organisation">NATO</acronym></li>

	<li>abbr <acronym title="a další">et al.</acronym></li>

	<li>náhrady™ nebo® za pøíslušné ©entity</li>

	<li>šipky &#8592; a &#8594; a &#8592; &gt;</li>

	<li>tøi teèky…</li>

	<li>zachování HTML entit &amp;</li>

	<li>a spousta dalšího :-)</li>
</ul>
